sys/amd64/amd64/pmap.c

```diff
@@ -7146,7 +7145,7 @@ pmap_enter_quick_locked(pmap_t pmap, vm_offset_t va, vm_page_t m,
 {
 	pt_entry_t newpte, *pte, PG_V;
 
-	KASSERT(va < kmi.clean_sva || va >= kmi.clean_eva ||
+	KASSERT(!VA_IS_CLEANMAP(va) ||
```

Comment:
I really should pull this out and upstream it to FreeBSD.

Reply:
Do you want me to split this out so you can cherry-pick or leave it for now?

Reply:
I split the commit, so hopefully upstreaming it is easier.